src/Sylius/Bundle/AddressingBundle/Form/Type/CountryChoiceType.php

```diff
@@ -64,6 +68,7 @@ public function configureOptions(OptionsResolver $resolver): void
                 'enabled' => true,
                 'label' => 'sylius.form.address.country',
                 'placeholder' => 'sylius.form.country.select',
+                'filter' => null,
```

Comment:
Let's name it `choice_filter` to make it more self-explaining and consistent with `ZoneCodeChoiceType`.